### PR TITLE
Introduced new 'externalOutputDir' config flag

### PR DESCRIPTION
**Pull request by itsmejoeeey**
Introduced this flag to allow users to define their own external images download directory. This comes in handy when using the plugin without `next export`.

See further discussion here:
https://github.com/dc7290/next-export-optimize-images/issues/105#issuecomment-1225750849

### CODE REVIEW

**Comment by dc7290** [comment]
@itsmejoeeey 
Thanks for creating the PR!

I have one request, please adapt the commit message to the rules of this repository!
As it is now, we cannot put the name itsmejoeeey in the release notes.😭

Therefore, I would like to ask you to revert the current commit and then make the same commit again.
You can use `git-cz` to do this easily on your local terminal.
Here is a brief explanation of how to do it.
https://github.com/dc7290/next-export-optimize-images/blob/main/CONTRIBUTING.md

**Comment by itsmejoeeey** [comment]
I forgot `--include=dev` when I initially ran `npm install` so the hooks never ran. This look better?

**Review by dc7290 — APPROVED** [review]
Thanks!
LGTM!

**Comment by dc7290** [comment]
@itsmejoeeey 

> I forgot --include=dev when I initially ran npm install so the hooks never ran. This look better?

I see.
I'm not too familiar with the `npm` spec, partly because I usually use `yarn`.
Did something go wrong?
If you don't mind me asking, I'd love to know!

File: src/image.tsx [hunk 1]
```diff
@@ -53,6 +53,9 @@ const exportableLoader: ImageLoader = ({ src: _src, width, quality }) => {
   const outputDir = `/${
     config.imageDir ? config.imageDir.replace(/^\//, '').replace(/\/$/, '') : '_next/static/chunks/images'
   }`
+  const externalOutputDir = `/${
+    config.externalImageDir ? config.externalImageDir.replace(/^\//, '').replace(/\/$/, '') : '_next/static/media'
+  }`
```
Comment (dc7290) [inline src/image.tsx:58]: Looks like the leading slash is unnecessary!

**Review by dc7290 — CHANGES_REQUESTED** [review]

File: src/image.tsx (hunk 1 above)
Comment (itsmejoeeey) [reply src/image.tsx:58]: Apologies for taking a while to get back to this, was on holiday.

**Review by dc7290 — APPROVED** [review]
Nice!!
LTGM!

**Pull request merged**